_docs/style/colors.md

```diff
@@ -10,82 +10,553 @@ status: complete
+
+#### Health status states
+
+In the default state, the critical, warning, and ok statuses map to the 500 band of red, yellow, and green, respectively.
+
+In the hover state, use the 700 band of red, yellow, and green.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Health status states"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/color-palette/health-status-states.png" width="484"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+### Non-health status
+
+Sometimes it is important to communicate the status of an entity, without also communicating urgency or criticality. The purpose of a non-health status indicator is to draw the attention of the user so that they can explore further, when desired.
+
+For example, use a badge or New! tag to indicate new items are available for review. These status labels use Orange 500.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Badge and New! tags"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/color-palette/non-health-status.png" width="1060"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Status labels
+
+You can also use status labels as a way to communicate the status of an object, for example, a support ticket. Unlike health status which is a closed system that prescribes that you use specific icons and colors to convey meaning, status labels are non-use case specific, and you can use them as you see fit. Review your system of statuses to determine which label to use.
+
+{% endcolumn %}
+
+{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+{% figure [caption:"Status labels"] [class:"image bg-light border"] %}
+
+<img src="{{site.url}}/assets/images/style/color-palette/status-labels.png" width="1008"/>
+
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+
+#### Alert banners
+
+Use an alert banner to communicate that a service has been or could be negatively impacted.
+
+Do not construct an alert banner to display the status of a particular object or entity. An alert banner should apply to the larger system. For example, use the red alert banner to communicate that a server is unavailable and use the green alert banner to communicate that the server is back online.
```

Comment:
these 3 sentences are directly contradictory. lolz.  Do we need to have alert bars on this page at all? I would prob cut them. Instead we can focus on the 100 band as bg color and list the components that use it, including alert bars. 

this way we're not realing with WTU or BP for any specific component, only the use of co
lor.